### DISCUSSION TITLE
Tested on Noxon and Denon

### DISCUSSION THREAD

**Discussion by xaanur** [coffeegreg/YTuner#30] [Show and tell]
Running YTuner on Ubuntu Server 24 LTS in Hyper-V VM

Tested and working fine on:
Denon RCD-N9 CEOL
Denon S-32
Noxon iRadio 300

btw thank you for the great work, YTuner is a masterpiece

**Comment by coffeegreg** [comment]
I'm glad to know that YTuner works properly for you and thank you for appreciating my work. :+1: 
I'll update the "Confirmed working" list. :rocket:
